fvm/environment/facade_env.go

```diff
@@ -157,6 +158,33 @@ func NewScriptEnvironment(
 		})
 }
 
+// This is mainly used by the emulator and cadence tools testing.
+func NewEmulatorScriptEnvironment(
```

Comment:
looks like we have some command line tool in flow-go which could also switch to this.  I'll rename this function and update the cmd line tool while i'm at it.

Reply:
hmm.  looks like cmd line change is a bit involved.  I'll only update the function name in this pr.